frontend/src/PlaylistDisplay/PlaylistDetail.tsx

```diff
@@ -19,15 +19,14 @@ export default function PlaylistDetail(){
             headers:{
                 "Authorization": "Bearer"+ localStorage.getItem("jwt")
             }})
-        .then(response => {return response.json()})
-        .then(responseBody  => {
-            if(responseBody){
-                return responseBody;   
+        .then(response => {
+            if(!(response.status === 404)){
+                return response.json()
             }
             throw new Error("There is no Playlist with the requested id");
-         })
+        })
         .then(responseBody => {setPlaylist(responseBody); setReadyToRender("yes");})
```

Comment:
what about: .then(responseBody : Array<responseBody>?

Reply:
It works also without defining the respondBody type as Array